### PR TITLE
Fixes #132: Moved scripts to script folder

### LINKED ISSUE #132: Move the script to script folder
- Issue type: Feature request 

### Short description

Moving the script like `debuid.sh` to the script folder.

### PR DESCRIPTION
**Pull request by vaibhavsingh97**
Fixes #132 
### Short description
- Moved all the `*.sh` files to Script folder

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by meets2tarun** [comment]
Can you please check the Travis log result and resolve the issue?
https://travis-ci.org/fossasia/meilix/builds/280630365#L8121

**Comment by xeon-zolt** [comment]
simply moving the scripts like build and chroot is not required and should not be done

**Comment by vaibhavsingh97** [comment]
@xeon-zolt I pushed the changes wthout chroot but Travis build is failing so I added

**Comment by vaibhavsingh97** [comment]
@meets2tarun I will fix ;)

**Comment by tabesin** [comment]
Just a minor issue. 
mkdir: cannot create directory ‘/tmp/./scripts/build.sh.tempmount.3531’: No such file or directory

**Comment by vaibhavsingh97** [comment]
@tabesin Only solution is to keep the `build.sh` outside the scripts folder. Should i proceed?

**Comment by meets2tarun** [comment]
@vaibhavsingh97 Check that possibility too!

**Comment by vaibhavsingh97** [comment]
@meets2tarun @tabesin  Should I need to squash?

File: .travis.yml [hunk 1]
```diff
@@ -23,6 +23,7 @@ include:
 sudo: required
 
 script:
+  - chmod +x ./build.sh
```
Comment (xeon-zolt) [inline .travis.yml]: its already in +x

Comment (vaibhavsingh97) [reply .travis.yml]: @xeon-zolt But yesterday, it gave me an error os i added this

**Pull request merged**